Sources/FunctionalTableData/CollectionView/FunctionalCollectionData+UICollectionViewDelegate.swift

```diff
@@ -113,5 +113,27 @@ extension FunctionalCollectionData {
 			
 			return proposedIndexPath
 		}
+		
+		@available(iOS 13.0, *)
+		public func collectionView(_ collectionView: UICollectionView, contextMenuConfigurationForItemAt indexPath: IndexPath, point: CGPoint) -> UIContextMenuConfiguration? {
+			let section = data.sections[indexPath.section]
+			let row = section.rows[indexPath.row]
```

Comment:
Could there ever be a case where any of these 2 could fail and throw an `Index out of range`?

Reply:
It is unlikely, but possible that it could happen if we are generating a context menu while a render occurs. I've added a guard to avoid this.